ui/src/routes/_layout/organizations/$orgId.products.$productId.repositories.$repoId.create-run.tsx

```diff
@@ -987,7 +1008,62 @@ const CreateRunPage = () => {
                 <AccordionItem value='evaluator' className='flex-1'>
                   <AccordionTrigger>Evaluator</AccordionTrigger>
                   <AccordionContent>
-                    No job configurations yet implemented for Evaluator.
+                    <FormField
+                      control={form.control}
+                      name='jobConfigs.evaluator.ruleSet'
+                      render={({ field }) => (
+                        <FormItem>
+                          <FormLabel>Evaluator rules</FormLabel>
+                          <FormControl>
+                            <Input {...field} />
+                          </FormControl>
+                          <FormDescription>
+                            The path to the rules file which is resolved from
+                            the configured configuration source. If this is left
+                            empty, the default path from ORT will be used.
```

Comment:
BTW, what are the convention to wrap (or not) such texts? Should we maybe start to follow the one-sentence-per-line rule that we aim to follow for Markdown?

Reply:
Prettier automatically formats the files and wraps the text based on something, so I'd just let it do it.

Reply:
This part of ["Why Prettier?"](https://prettier.io/docs/en/why-prettier#building-and-enforcing-a-style-guide) may be relevant. ;)

> By far the biggest reason for adopting Prettier is to stop all the on-going debates over styles. [It is generally accepted that having a common style guide is valuable for a project and team](https://www.smashingmagazine.com/2012/10/why-coding-style-matters/) but getting there is a very painful and unrewarding process. People get very emotional around particular ways of writing code and nobody likes spending time writing and receiving nits.

Reply:
I never do line breaking manually in the code files, as Prettier does far better job for me!

Reply:
> nobody likes spending time writing and receiving nits.

That's absolutely not true 🤣